modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModalImportObjectDefinition.tsx

```diff
@@ -69,7 +69,12 @@ export default function ModalImportObjectDefinition({
 
 	const handleImport = async (formData: FormData) => {
 		try {
-			await API.save(importObjectDefinitionURL, formData, 'POST');
+			await API.save({
+				item: formData,
+				method: 'POST',
+				returnValue: false,
```

Comment:
@igor-franca The default returnValue is already false, I don't think we need to pass this property here, right?

Reply:
Well noted, thanks! 👍🏾